### PR TITLE
[close #283] add it for kill downstream

### LINKED ISSUE #283: cdc stuck when kill downstream cluster
### 1. Describe the bug
cdc stuck when kill downstream cluster

### 2. Minimal reproduce step (Required)
1. start upstream & downstream cluster
2. create changefeed
3. input data into upstream cluster
4. kill downstream cluster

### 3. What did you see instead (Required)
1. cdc stuck
### 4. What did you expect to see? (Required)
1. report error
### 5. What is your migration tool and TiKV version? (Required)

- TiKV:
- TiKV CDC: master

### PR DESCRIPTION
**Pull request by zeminzhou**
Signed-off-by: zeminzhou <zhouzemin@pingcap.com>

### What problem does this PR solve?

Issue Number: close #283

Problem Description: **TBD**

### What is changed and how does it work?
Add integration test case for kill downstream

### Code changes

- No code

### Check List for Tests

This PR has been tested by at least one of the following methods:
- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)

### Side effects

- No side effects

### Related changes

- No related changes

### CODE REVIEW

**Comment by zeminzhou** [comment]
PTAL~

**Review by pingyu — APPROVED** [review]
LGTM~

File: cdc/tests/integration_tests/kill_downstream/run.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+	down_pd_pid=$(ps -aux | grep "pd-server" | grep "down_pd" | awk '{print $2}' | head -n1)
+	down_tikv_pid=$(ps -aux | grep "tikv-server" | grep "tikv_down" | awk '{print $2}' | head -n1)
+
+	# stop downstream
+	kill -19 $down_pd_pid
+	kill -19 $down_tikv_pid
+
+	# Wait for cdc to retry to create tikv sink
+	sleep 180
```
Comment (pingyu) [inline cdc/tests/integration_tests/kill_downstream/run.sh:39]: The `180s` would be too long for efficiency. The integrated tests have already cost too much time.
Maybe we can find some way to speed up the retry to achieve similar result later. The test for "real scenario" can be done in e2e tests.

**Review by haojinming — APPROVED** [review]
LGTM

**Pull request merged**